CLDConfig/cld_steer.py

```diff
@@ -36,8 +36,8 @@
 SIM.macroFile = ""
 ## number of events to simulate, used in batch mode
 SIM.numberOfEvents = 0
-## Outputfile from the simulation,only lcio output is supported
-SIM.outputFile = "dummyOutput.slcio"
+## Outputfile from the simulation
+SIM.outputFile = "CLD_SIM.root"
```

Comment:
better to make it obvious that its an edm4hep file

```suggestion
SIM.outputFile = "CLD_SIM.edm4hep.root"
```

Reply:
Hi Leonhard,

Indeed, actually this was mostly to remove the "only lcio output is supported" comment. But having two dots in the file name can be annoying. And do we really need to specify that it is edm4hep? I plan to have file names like `electron_gun_10GeV_IDEA_SIM.root`, `electron_gun_10GeV_ALLEGRO_RECO.root`, etc, in other workflows. No strong opinion though because as you say we will most of the time overwrite it, will do how you prefer. Let me know.

Reply:
We do not need to do it, but I would prefer it. I think it is also useful to make the users aware of the fact that this specific format of root file has a name. 

If more than one dot annoys you I am also happy with `_edm4hep.root` but I thought we are trying to transition to the version with two dots. Is there a specific issue with more than one dot? This should be somewhat standard for "nested" formats see e.g. `.tar.gz` files.

Reply:
Fair enough, as long as we have edm4hep output by default and do not claim that only lcio is supported, I am happy :-)